test.py

```diff
@@ -1654,18 +1309,63 @@ async def find_tests(options: argparse.Namespace) -> None:
                 suite = TestSuite.opt_create(f, options, mode)
                 await suite.add_test_list()
 
-    if not TestSuite.test_count():
-        if len(options.name):
-            print("Test {} not found".format(palette.path(options.name[0])))
-            sys.exit(1)
-        else:
-            print(palette.warn("No tests found. Please enable tests in ./configure.py first."))
-            sys.exit(0)
-
     logging.info("Found %d tests, repeat count is %d, starting %d concurrent jobs",
                  TestSuite.test_count(), options.repeat, options.jobs)
     print("Found {} tests.".format(TestSuite.test_count()))
 
+def run_pytest(options: argparse.Namespace, run_id: int):
+    tmp_dir = pathlib.Path(options.tmpdir)
+    allure_dir = tmp_dir / 'allure'
+    expression = ''
+    if options.name:
+        expression += ' '.join(options.name)
+    expression += f'{"and not ".join(options.skip_patterns)}' if options.skip_patterns else ''
+    modes = ' '.join([f'--mode={mode}' for mode in options.modes])
+    args = [
+        'python',
+        '-m',
+        'pytest',
+        "-s",  # don't capture print() output inside pytest
+        '--color=yes',
+        "--log-level=DEBUG",  # Capture logs
+        '-v',
+        "--junit-xml={}".format(tmp_dir / f'pytest_cpp_{run_id}.xml'),
+        "-rs",
```

Comment:
why -rs ? not just -r or -rA

Reply:
Its parameters used for PythonSuite, so just copy paste